### PR TITLE
Initial structure

### PR DESCRIPTION
**Pull request by tomahawk28**
- Initial CircleCI exporter structure
- No metrics yet.

### CODE REVIEW

**Comment by MarcelMue** [comment]
Please set up CI correctly for this Repo. I don't think this would build successfully because of the missing flag structure.

**Comment by tomahawk28** [comment]
@MarcelMue It has flag structure on it already, https://github.com/giantswarm/circleci-exporter/blob/dd007aeb7a243d7b63c3ed9352e33ffc98ac39cd/main.go#L57. 
I mostly copy circleCI setup from giantswarm/github-exporter. should I build more structure than this?

**Comment by MarcelMue** [comment]
@tomahawk28 It looks like you registered this project to circle 2 hours ago.

Can you push a commit into this branch to see if it builds correctly now?

**Review by MarcelMue — APPROVED** [review]
LGTM <3

File: service/collector/collector.go [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package collector
+
+const (
+	namespace = "github_exporter"
```
Comment (TheoBrigitte) [inline service/collector/collector.go]: Remove this file if you don't need it.

**Review by TheoBrigitte — CHANGES_REQUESTED** [review]
LGTM, but I don't think you need `service/collector/collector.go`

File: service/collector/collector.go (hunk 1 above)
Comment (tomahawk28) [reply service/collector/collector.go]: Thanks, I didn't notice that.

**Review by TheoBrigitte — APPROVED** [review]

**Pull request merged**